modules/core/src/test/java/org/apache/ignite/internal/DiagnosticLogForPartitionStatesTest.java

```diff
@@ -105,20 +166,33 @@ private void doTest(CacheConfiguration<Object, Object> cacheCfg, boolean msgExp)
 
         node1.cluster().active(true);
 
-        IgniteCache<Object, Object> cache = node1.getOrCreateCache(cacheCfg);
+        List<IgniteCache<Integer, Integer>> caches = new ArrayList<>(cfgs.size());
```

Comment:
Try to use streams everywhere:)

Reply:
Why do you think so? What is the reason to use streams everywhere? :)